### PR TITLE
Add toggle for Docker install/deinstall for baseline test

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Add a -d switch to the test_podman_baseline.sh script.  When used, the script will NOT install nor remove Docker from the test machine.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: test/test_podman_baseline.sh [hunk 1]
```diff
@@ -7,20 +7,30 @@
...
 #######
-# See if we want to stop on errors or not.
+# See if we want to stop on errors and/or install and then remove Docker.
 #######
 showerror=0
-while getopts "e" opt; do
+installdocker=0
+while getopts "de" opt; do
     case "$opt" in
+    d) installdocker=1
```
Comment (baude) [inline test/test_podman_baseline.sh]: I personally would prefer a test for docker and or an active daemon rather than a switch.  just my way ...

Comment (TomSweeneyRedHat) [reply test/test_podman_baseline.sh]: Yeah, I just went with the KISS method.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**